<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Comment</class>
 <widget class="QWidget" name="Comment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/user.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="commentTitle">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>320</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>commentTitle</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="commentText">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>320</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>commentText</string>
   </property>
  </widget>
  <widget class="QLabel" name="commentTime">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2017-09-15 12:30:09</string>
   </property>
  </widget>
  <widget class="QLabel" name="commentUser">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
